test/azure-pipelines.yml

```diff
@@ -156,7 +157,7 @@ jobs:
     displayName: 'Fetch build script'
 
   - script: |
-      export ITK_PACKAGE_VERSION=$(ITKGitTag)
+      export ITK_PACKAGE_VERSION=$(ITKGitTag)$(ITKPythonPost)
```

Comment:
Do these changes apply to the `ITKModuleTemplate` as well?

Reply:
Yes! :-) PR here: https://github.com/InsightSoftwareConsortium/ITKModuleTemplate/pull/64